includes/functions/content-sync.php

```diff
@@ -144,7 +144,7 @@ function send_track_event( $tracker, $post, $action ) {
 	 * 
 	 * @return {array} Tracking data to send.
 	 */
-	$data = apply_filters_ref_array( 'sophi_tracking_data', array( $data, &$tracker, $post, $action ) );
+	$data = apply_filters_ref_array( 'sophi_tracking_request_data', array( $data, &$tracker, $post, $action ) );
```

Comment:
Could similarly go with `sophi_cms_tracking_request_data` since this is the portion of the plugin where we send post publishing/update events, correct?  Would be separate from the javascript tracking that happens on the site frontend, yeah?

Reply:
in which case maybe the other one gets updated to `sophi_js_tracking_request_data`?

Reply:
@jeffpaul great idea. I also updated the second filter `sophi_cms_tracking_result` for consistency